### PR TITLE
small corrections applied to all versions

### PR DESCRIPTION
**Pull request by ericharmeling**
This includes the changes made in #5128 and #5129, but to all versions.

### CODE REVIEW

File: v19.1/architecture/replication-layer.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Because CockroachDB serves reads from a range's leaseholder, it benefits your cl
 
 {{site.data.alerts.callout_info}}
 
-This feature is also called [Follow-the-Workload](http://localhost:4000/docs/stable/demo-follow-the-workload.html) in our documentation.
+This feature is also called [Follow-the-Workload](https://www.cockroachlabs.com/docs/stable/demo-follow-the-workload.html) in our documentation.
```
Comment (rmloveland) [inline v19.1/architecture/replication-layer.md]: This link should be relative, i.e.

```
[Follow the Workload](../demo-follow-the-workload.html)
```

That way it always refers to the same version as this page.

File: v19.2/architecture/replication-layer.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Because CockroachDB serves reads from a range's leaseholder, it benefits your cl
 
 {{site.data.alerts.callout_info}}
 
-This feature is also called [Follow-the-Workload](http://localhost:4000/docs/stable/demo-follow-the-workload.html) in our documentation.
+This feature is also called [Follow-the-Workload](https://www.cockroachlabs.com/docs/stable/demo-follow-the-workload.html) in our documentation.
```
Comment (rmloveland) [inline v19.2/architecture/replication-layer.md]: Same comment as previously re: using a relative link.

File: v19.1/architecture/replication-layer.md (hunk 1 above)
Comment (ericharmeling) [reply v19.1/architecture/replication-layer.md]: Done.

File: v19.2/architecture/replication-layer.md (hunk 2 above)
Comment (ericharmeling) [reply v19.2/architecture/replication-layer.md]: Done.

**Pull request merged**